front/ui/ui-charts/src/spaceTimeChart/components/SpaceTimeChart.tsx

```diff
@@ -222,7 +222,7 @@ export const SpaceTimeChart = (props: SpaceTimeChartProps) => {
             {!hideGrid && (
               <>
                 <SpaceGraduations />
-                <TimeGraduations />
+                <TimeGraduations mouseX={mouseContext.position.x} isHover={mouseContext.isHover} />
```

Comment:
As `MouseContext.Provider` wrap `TimeGraduations` we can get `position` and `isHover` directly inside `TimeGraduations` using `React.useContext`

```typescript
const { position, isHover } = useContext(MouseContext);
```


```suggestion
                <TimeGraduations />
```